birthday problem done

hyunseogu opened the pull request.

hyunseogu committed:
feat: Create birthday-problem.py

hyunseogu committed:
feat: Deleted birthday-problem.py

to chage file type, delete .py file

hyunseogu committed:
feat: Create birthday-problem.md

To explain birthday problem, create birthday-problem.md

hyunseogu committed:
feat: add the information about birtday problem

Explain what is birthday problem and how to caculate probablity meet
someone with the same birthday

jaechanjo commented on birthday-problem.md:
```diff
@@ -0,0 +1,38 @@
+
+두 사람의 생일이 같지 않으려면 사람1의 생일이 사람2의 생일을 제외한 다른 날짜여야한다. 즉 사람 1의 생일을 제외한 364일중 하루가 된다.
+
+365/365 x 364/365 = 99.72%로 생일이 같지 않을 확률을 가진다.
```
오! 이런 공식이었군요! 위의 맥락을 맞춰주는 차원에서 [1 - (1/365 X 1/364_생일이 같지 않은 확률)] X 100(%) = 99.72%로 써주면 더 좋을 것 같습니다!

hyunseogu replied:
아하 그러면 전부 생일이 같을 확률로 수정해보겠습니다

jaechanjo replied:
좋습니다!

hyunseogu committed:
feat: Modified how to calculate the probability

changed calculating way, from probability of having different birthday
to the same
